### PR TITLE
Image gallery crash fix

### PR DESCRIPTION
**Pull request by jackahl**
Fix for  #1722 

Listingbody now detects whether it uses a the Gallery template and if so sends a request to the querystring endpoint to retrieve the fullobjects. This leads to the image gallery no longer crashing when no criteria are set (because than the block does not retrieve the fullobjects needed for that template)

### CODE REVIEW

File: src/components/manage/Blocks/Listing/ListingBody.jsx [hunk 1]
```diff
@@ -68,9 +87,11 @@ const ListingBody = ({ data, properties, intl, path, isEditMode }) => {
     );
   }
 
+  console.log(listingItems);
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/Listing/ListingBody.jsx:90]: console.log

**Comment by jackahl** [comment]
@tisto can you check why cypress fails here? I dont really understand the trace.

**Comment by tisto** [comment]
@jackahl did you try to run the Cypress tests locally? This is a regression on the listing block sort functionality.

File: src/components/manage/Blocks/Listing/ListingBody.jsx (hunk 1 above)
Comment (tisto) [reply src/components/manage/Blocks/Listing/ListingBody.jsx]: @tiberiuichim I recall that I added an eslint rule to report console.log statements. No idea why this does not kick in...

**Pull request merged**